lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -856,21 +856,34 @@ function dosomething_reportback_mbp_request($entity) {
 
   if (module_exists('dosomething_user')) {
     $account = user_load($entity->uid);
+    $node = node_load($entity->nid);
+    $language = dosomething_global_get_language($account, $node);
+
+    $title = $entity->node_title;
```

Comment:
With the presence of `node_load($entity->nid);` above should the "node isn't loaded for some reason" be logged / reported as an error?